src/Compilers/CSharp/Portable/Binder/ExpressionVariableFinder.cs

```diff
@@ -282,6 +282,17 @@ public override void VisitRecursivePattern(RecursivePatternSyntax node)
             base.VisitRecursivePattern(node);
         }
 
+        public override void VisitListPattern(ListPatternSyntax node)
+        {
+            TFieldOrLocalSymbol variable = MakePatternVariable(null, node.Designation as SingleVariableDesignationSyntax, _nodeToBind);
+            if ((object)variable != null)
```

Comment:
```suggestion
            if (variable is not null)
```

Reply:
This is self-consistent with the file and I prefer to not make code style changes to avoid making the PR larger than it needs to be.